index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
Please add a descriptive README file to your project, to add a professional README file click here https://github.com/microverseinc/readme-template

Reply:
- [ ] Good comment @Josphat205, however, I would suggest you explain the reason for adding the professional `readme` file. For instance how it helps in describing the projects to viewers of the repo and makes the repo look professional.